phpstan/stubs/image.stub

```diff
@@ -115,17 +115,6 @@ function imagegif(\GdImage $image, $to = null): void {}
 function imagewebp(\GdImage $image, $to = null, int $quality = 80): void {}
 
 
-/**
- * @param \GdImage $image An image, returned by one of the image creation functions, such as imagecreatetruecolor.
- * @param float $angle Rotation angle, in degrees. The rotation angle is interpreted as the number of degrees to rotate the image anticlockwise.
- * @param int $bgd_color Specifies the color of the uncovered zone after the rotation
- * @param int $dummy This parameter is unused.
- * @return \GdImage Returns an image for the rotated image.
- * @throws ImageException
- */
-function imagerotate(\GdImage $image, float $angle, int $bgd_color, int $dummy = 0): \GdImage {}
```

Comment:
Instead of deleting that one, just comment it please, same resonnning as above.

Reply:
this one was deleted, because its duplicate function definition, if you can expand to see the full file, then you will see it.

Reply:
Good spot, my mistake :+1: